Assets/Mirror/Examples/Chat/Scripts/Player.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mirror.Examples.Chat
+{
+    public class Player : NetworkBehaviour
+    {
+        [SyncVar]
+        public string playerName;
+
+        public ChatWindow chatWindow => ((ChatNetworkManager)NetworkManager.singleton).chatWindow;
+
+        [Command]
+        public void CmdSend(string message)
+        {
+            // Restrict to ASCII printable characters by stripping anything outside the range from space through tilde
+            string cleanMessage = System.Text.RegularExpressions.Regex.Replace(message.Trim(), @"![ -~]", "");
```

Comment:
What about chinese, japanese, russian, korean, greek, latin characters?
what about emoji characters?

Why are we restricting the character set to only ASCII?